### PR TITLE
Preparing new repository

### PR DESCRIPTION
**Pull request by janbj**
Created new GitHub repository and added tools and integrations for static analysis.

### CODE REVIEW

**Comment by janbj** [comment]
Some checks are failing. I believe this is because the JacocoReport is empty and the CFN template contains no resources (yet).

File: config/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+      <property name="allowMissingReturnTag" value="true"/>
+      <property name="allowedAnnotations" value="Override, Test"/>
+      <property name="tokens" value="METHOD_DEF, CTOR_DEF, ANNOTATION_FIELD_DEF"/>
+    </module>
+    <module name="MissingJavadocMethod">
+      <property name="scope" value="public"/>
+      <property name="minLineCount" value="2"/>
+      <property name="allowedAnnotations" value="Override, Test"/>
+      <property name="tokens" value="METHOD_DEF, CTOR_DEF, ANNOTATION_FIELD_DEF"/>
+    </module>
```
Comment (axthosarouris) [inline config/checkstyle/checkstyle.xml:299]: remove the missing javadoc config, it creates more problems that it solves. we should update the template project

**Review by axthosarouris — COMMENTED** [review]
I would suggest adding a gradle.properties file with the following content:

```
org.gradle.parallel=true
org.gradle.caching=true
org.gradle.console=verbose
org.gradle.warning.mode=all
```

**Comment by axthosarouris** [comment]
missing codacy secret in github actions?

**Comment by axthosarouris** [comment]
why is it failing?

**Pull request merged**